packages/tools/src/tools/segmentation/BrushTool.ts

```diff
@@ -584,6 +584,33 @@ class BrushTool extends BaseTool {
       rightCursorInWorld,
     ];
 
+    const activeStrategy = this.configuration.activeStrategy;
+
+    // Note: i don't think this is the best way to implement this
+    // but don't think we have a better way to do it for now
+    if (activeStrategy.startsWith('THRESHOLD')) {
```

Comment:
Instead of this, create a new active strategy callback that returns configuration values, can call that method and get the data from the response.  That way you can change the inner circle size based on whatever the strategy decides.

Reply:
Done